pkg/collector/corechecks/sbom/check.go

```diff
@@ -173,13 +180,23 @@ func (c *Check) Run() error {
 			c.processor.processContainerImagesRefresh(c.workloadmetaStore.ListImages())
 		case <-hostPeriodicRefreshTicker.C:
 			c.processor.processHostRefresh()
+		case <-metricTicker.C:
+			c.sendUsageMetrics()
```

Comment:
Would it be possible to send metrics from the `Run` method?

Reply:
Do you mean directly calling it from `Run` instead of calling a submethod from `Run`

Reply:
Oops, I missed the fact that this is already inside Run, and a long-running check to boot. Never mind then.